spec/models/referee_interface/reference_feedback_form_spec.rb

```diff
@@ -36,5 +36,23 @@
       expect(application_form).not_to be_application_references_complete
       expect(application_form.application_choices).to all(be_awaiting_references)
     end
+
+    it 'progresses the application choices to the "awaiting_provider_decision" status once all references have been received if edit_by expired' do
```

Comment:
Does this spec now duplicate `services/receive_reference_spec.rb`? (not just the bit changed here but the two preceding specs too)